README.md

```diff
@@ -62,7 +61,7 @@ const configuration: APIConfigurationParameters = {
 }
 
 const client = new UploadsApiClient(configuration)
-const parameters: UploadsApiModel.UploadsApiCreateUploadDestinationForResourceRequest = {
+const parameters = {
```

Comment:
Why not just pass them straight to the method? That way the method will have type support already.

Reply:
@moltar : Thanks for your suggestion. I have fixed them. 👍